R/addLDA.R

```diff
@@ -0,0 +1,105 @@
+#' Latent Dirichlet Allocation
+#'
+#' These functions perform Latent Dirichlet Allocation on data stored in a 
+#'  \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#' 
+#' @param x a \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#'   
+#' @param k Integer, number of latent vectors/topics 
+#'   (By default: \code{2}).
+#' 
+#' @param name The name to be used to store the result in the reducedDims of the
+#'  output (By default: \code{name = "LDA"}).
+#'  
+#' @param assay.type a single \code{character} value for specifying which
+#'   assay to use for LDA ordination.
+#' 
```

Comment:
Character scalar. Specifies which assay to use for LDA ordination. (Default: \code{"counts"})